test/SILOptimizer/sil_combine_global_addr.sil

```diff
@@ -0,0 +1,64 @@
+// RUN: %target-sil-opt -assume-parsing-unqualified-ownership-sil -enforce-exclusivity=none -enable-sil-verify-all %s -sil-combine -verify-skip-unreachable-must-be-last -devirtualizer | %FileCheck %s
+
+// Test to see if concrete type can be propagated from
+// global_addr in sil_combiner.
+sil_stage canonical
+
+import Builtin
+import Swift
+protocol SomeProtocol {
+  func foo() -> Int
+}
+class SomeClass  : SomeProtocol {
+  init()
+  func foo() -> Int
+}
+sil hidden [thunk] [always_inline] @foo_ : $@convention(witness_method:SomeProtocol) (@in_guaranteed SomeClass) -> Int {
```

Comment:
Please provide a failing test case where we have multiple init_existential_addr. I would just do this

BB1 (global) -> (BB2, BB3) where init existential_addr is -> BB4 the rest.